R/PipeOpOVR.R

```diff
@@ -93,15 +93,16 @@ PipeOpOVRSplit = R6Class("PipeOpOVRSplit",
       list(as.Multiplicity(private$.splittask(inputs[[1]], self$state$levels)))
     },
     .splittask = function(task, levels) {
+      new_target = paste0(task$target_names, "_tmp")
```

Comment:
is it necessary to rename the target?

Reply:
woudl `convert_task(ntsk$cbind(truthcol), ...` also work?

Reply:
(without the rename)

Reply:
then we'd have the same column twice in the Task.
If we would keep the same name, we would rely on the slightly strange behaviour mlr3 exhibits once two columns with the same name exist in a task (in different cbinded backends) where one column is not reachable anymore through 
`$data()`. 
So we could not rename, but this would perhaps lead to other issues.

Reply:
Is overwriting columns with new data using `$cbind()` not the "official" behaviour of mlr3?

Reply:
I don't know what is "official" there :), but I guess this could be intended behavior.
Feel free to re-write and merge. The convert_task branch will hopefully be merged tomorrow.